.appveyor.yml

```diff
@@ -1,5 +1,17 @@
 # Documentation: https://wiki.unvanquished.net/wiki/Continuous_integration
 
+only_commits:
+  files:
+    - .appveyor.yml
+    - appveyor/
```

Comment:
There is no `appveyor/` in the Daemon repo.

Reply:
Oops, removed.